examples/simple-file-server/README.md

```diff
@@ -0,0 +1,55 @@
+# Simple File Server
+
+## Overview
+
+This guide explains how to put and get a file to/from an UDP server using Ballerina.
+The below are the detailed explanations of each of the steps of this example.
+
```

Comment:
Maybe we can have a section called `Implementation` and explain how this server/client works. It seems we have introduced our protocol by adding the sequence number. That way we can remove the comments on the code which makes the code less readable.

Reply:
Maybe we can have a small diagram too.

Reply:
Added the content with a diagram.